components/CustomListItem/CustomListItem.js

```diff
@@ -41,7 +41,7 @@ const CustomListItem = React.memo(({
 
                             <Icon
                                 name="chevron-right"
-                                color="#000000"
+                                color="#050038"
```

Comment:
Maybe there we can use predefined colour?

Reply:
CHanged to predefined colour.